tools/audio-codec-cli/src/feedback.rs

```diff
@@ -94,3 +99,199 @@ pub async fn feedback(args: StaticArgs) -> anyhow::Result<()> {
     println!("Done!");
     Ok(())
 }
+
+pub async fn echo(args: StaticArgs) -> anyhow::Result<()> {
+    let host = cpal::default_host();
+    let input_device = host.default_input_device().unwrap();
```

Comment:
Is it safe to unwrap here instead of returning an error? Would it be consider a bug if this do panic?

Reply:
considering that this is a test tool, i'm not at all concerned if it crashes. 

Still, it does seem silly to call unwrap() in a function that returns a Result. I'll change it.